homeassistant/components/fritz/switch.py

```diff
@@ -187,7 +188,9 @@ def port_entities_list(
         port_forwards_count,
     )
 
-    local_ip = get_local_ip()
+    local_ip = fritzbox_tools.hass.loop.run_until_complete(
```

Comment:
You can avoid the context switch by awaiting it in `async_setup_entry` and then passing it through to `all_entities_list`

Reply:
Done